go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/tailscale/squibble v0.0.0-20250108170732-a4ca58afa694
 	honnef.co/go/tools v0.6.1
 	modernc.org/sqlite v1.34.4
-	tailscale.com v1.83.0-pre.0.20250331211809-96fe8a6db6c9
+	tailscale.com v1.85.0-pre.0.20250523030259-7a5af6e6e7d4
```

Comment:
We usually update this repo to whatever version corp is using at the time, e.g., using tools/update-oss.sh. We do that because we depend on tailsql in corp, which may move ahead of the latest OSS release. This commit doesn't seem to match corp, did I miss something?